src/components/BlockCreateModal.vue

```diff
@@ -44,13 +52,18 @@
 
   const blockType = ref(props.providedBlockType)
   const capability = ref(props.capability ?? '')
+  const formId = randomId()
```

Comment:
What's this id on the block schema create form doing?

Reply:
So by adding an id to the form itself the submit button (which is outside of the form in the modal's actions) can submit the form. Something I've seen collin do that makes the modal look much nicer. The submit button has `:form-id="formId"`.

Reply:
ahhh ok that makes sense, i hadn't looked at that, thanks!